euth/reports/emails.py

```diff
@@ -0,0 +1,72 @@
+from django.contrib.auth import get_user_model
+from django.contrib.sites.models import Site
+from django.core import urlresolvers
+from django.db.models import ForeignKey
+from django.template import Context
+
+from euth.contrib import emails
+
+
+def send_email_to_moderators(request, report):
+
+    obj = report.content_object
+
+    if obj:
+        name = obj._meta.verbose_name.title()
+
+        try:
+            admin_url = urlresolvers.reverse("admin:%s_%s_change" %
+                                             (obj._meta.app_label,
+                                              obj._meta.model_name),
+                                             args=(obj.pk,))
+            admin_url = request.build_absolute_uri(
+                admin_url)
+        except:
+            admin_url = ''
+
+        moderators = get_user_model().objects.filter(is_superuser=True)
```

Comment:
Instead of using `get_user_model()` several times you can put this line at the top of the module (for example after the imports):
```python
User = get_user_model()
```
This allows you to write code like this:
```python
User.objects.filter(is_superuser=True)
```